app/models/container_template.rb

```diff
@@ -34,11 +34,23 @@ def instantiate(params, project = nil)
                                             :namespace => project
                                           },
                                           :objects    => objects,
-                                          :parameters => params)
+                                          :parameters => params_to_hashes(params))
     create_objects(processed_template['objects'], project)
     @created_objects.each { |obj| obj[:miq_class] = MIQ_ENTITY_MAPPING[obj[:kind]] }
   end
 
+  def params_to_hashes(params)
+    params.collect do |param|
+      {
+        :name     => param.name,
+        :value    => param.value,
+        :generate => param.generate,
+        :from     => param.from,
+        :required => param.required
+      }
+    end
+  end
+
```

Comment:
[Ugh](https://github.com/ManageIQ/guides/blob/7f5f49253b010e98cc489dc148a97539726f19e3/.rubocop_base.yml#L47-L48) I'm too lazy to start that battle again, nevermind.

Yes, we torched it.